pkg/apis/admissionregistration/types.go

```diff
@@ -118,7 +118,7 @@ const (
 // ExternalAdmissionHookConfiguration describes the configuration of initializers.
 type ExternalAdmissionHookConfiguration struct {
 	metav1.TypeMeta
-	// Standard object metadata; More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata.
+	// Standard object metadata; More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata.
```

Comment:
I've made git.k8s.io/$repo/$path take you to the right place, now, if you want to simplify :)